### ISSUE TITLE
`int_roundings` feature breaks the `num` crate

### ISSUE THREAD

**Issue by SkiFire13** [rust-lang/rust#88617]
This is due to the new `div_ceil` method that conflicts with the method with the same name in the `num::integer::Integer` trait. This matters for example when compiling the `num-bigint` crate, which now fails with this error:

<details><summary>Error</summary>

```
   Compiling num-bigint v0.4.1
error[E0658]: use of unstable library feature 'int_roundings'
   --> G:\Programmi\Rust\.cargo\registry\src\github.com-1ecc6299db9ec823\num-bigint-0.4.1\src\biguint.rs:398:45
    |
398 |                 let root_scale = extra_bits.div_ceil(&n64);
    |                                             ^^^^^^^^
    |
    = note: see issue #88581 <https://github.com/rust-lang/rust/issues/88581> for more information
    = help: add `#![feature(int_roundings)]` to the crate attributes to enable

error[E0308]: mismatched types
   --> G:\Programmi\Rust\.cargo\registry\src\github.com-1ecc6299db9ec823\num-bigint-0.4.1\src\biguint.rs:398:54
    |
398 |                 let root_scale = extra_bits.div_ceil(&n64);
    |                                                      ^^^^ expected `u64`, found `&u64`
    |
help: consider removing the borrow
    |
398 -                 let root_scale = extra_bits.div_ceil(&n64);
398 +                 let root_scale = extra_bits.div_ceil(n64);
    |

error[E0658]: use of unstable library feature 'int_roundings'
  --> G:\Programmi\Rust\.cargo\registry\src\github.com-1ecc6299db9ec823\num-bigint-0.4.1\src\biguint\convert.rs:70:10
   |
70 |         .div_ceil(&big_digit::BITS.into())
   |          ^^^^^^^^
   |
   = note: see issue #88581 <https://github.com/rust-lang/rust/issues/88581> for more information
   = help: add `#![feature(int_roundings)]` to the crate attributes to enable

error[E0308]: mismatched types
  --> G:\Programmi\Rust\.cargo\registry\src\github.com-1ecc6299db9ec823\num-bigint-0.4.1\src\biguint\convert.rs:70:19
   |
70 |         .div_ceil(&big_digit::BITS.into())
   |                   ^^^^^^^^^^^^^^^^^^^^^^^ expected `u64`, found reference
   |
   = note:   expected type `u64`
           found reference `&_`
help: consider removing the borrow
   |
70 -         .div_ceil(&big_digit::BITS.into())
70 +         .div_ceil(big_digit::BITS.into())
   |

error[E0658]: use of unstable library feature 'int_roundings'
   --> G:\Programmi\Rust\.cargo\registry\src\github.com-1ecc6299db9ec823\num-bigint-0.4.1\src\biguint\convert.rs:585:10
    |
585 |         .div_ceil(&u64::from(bits))
    |          ^^^^^^^^
    |
    = note: see issue #88581 <https://github.com/rust-lang/rust/issues/88581> for more information
    = help: add `#![feature(int_roundings)]` to the crate attributes to enable

error[E0308]: mismatched types
   --> G:\Programmi\Rust\.cargo\registry\src\github.com-1ecc6299db9ec823\num-bigint-0.4.1\src\biguint\convert.rs:585:19
    |
585 |         .div_ceil(&u64::from(bits))
    |                   ^^^^^^^^^^^^^^^^ expected `u64`, found `&u64`
    |
help: consider removing the borrow
    |
585 -         .div_ceil(&u64::from(bits))
585 +         .div_ceil(u64::from(bits))
    |

error[E0658]: use of unstable library feature 'int_roundings'
   --> G:\Programmi\Rust\.cargo\registry\src\github.com-1ecc6299db9ec823\num-bigint-0.4.1\src\biguint\convert.rs:613:10
    |
613 |         .div_ceil(&u64::from(bits))
    |          ^^^^^^^^
    |
    = note: see issue #88581 <https://github.com/rust-lang/rust/issues/88581> for more information
    = help: add `#![feature(int_roundings)]` to the crate attributes to enable

error[E0308]: mismatched types
   --> G:\Programmi\Rust\.cargo\registry\src\github.com-1ecc6299db9ec823\num-bigint-0.4.1\src\biguint\convert.rs:613:19
    |
613 |         .div_ceil(&u64::from(bits))
    |                   ^^^^^^^^^^^^^^^^ expected `u64`, found `&u64`
    |
help: consider removing the borrow
    |
613 -         .div_ceil(&u64::from(bits))
613 +         .div_ceil(u64::from(bits))
    |

Some errors have detailed explanations: E0308, E0658.
For more information about an error, try `rustc --explain E0308`.
error: could not compile `num-bigint` due to 8 previous errors
```

</details>

`rustc --version --verbose`:
```
rustc 1.56.0-nightly (371f3cd3f 2021-09-02)
binary: rustc
commit-hash: 371f3cd3fe523d0b398ed1db1620667c53ba7d02
commit-date: 2021-09-02
host: x86_64-pc-windows-msvc
release: 1.56.0-nightly
LLVM version: 13.0.0
```

**Comment by CryZe** [comment]
This is acceptable breakage, see: https://github.com/rust-lang/rust/issues/88581#issuecomment-912206298

**Comment by cuviper** [comment]
This was reported as rust-num/num-bigint#218, and fixed in 0.3.3 and 0.4.2.
